src/core/config/tmf.h

```diff
@@ -45,7 +45,7 @@
  */
 #ifndef OPENTHREAD_CONFIG_TMF_ADDRESS_CACHE_ENTRIES
 #if OPENTHREAD_CONFIG_BORDER_ROUTER_ENABLE
-#define OPENTHREAD_CONFIG_TMF_ADDRESS_CACHE_ENTRIES 256
+#define OPENTHREAD_CONFIG_TMF_ADDRESS_CACHE_ENTRIES 273
```

Comment:
Should we consider keeping at 256? The Thread Spec requirement is 232. With the snoop entry change, we should still satisfy that requirement.